prometheus.yml

```diff
@@ -0,0 +1,18 @@
+global:
+    scrape_interval: 15s
+    evaluation_interval: 15s
+
+scrape_configs:
+    - job_name: 'prometheus'
+      static_configs:
+          - targets: ['localhost:9090']
+    - job_name: 'testunit'
+      static_configs:
+        - targets: ['testunit']
+    - job_name: 'teste2e'
+      static_configs:
+        - targets: ['teste2e']
+    - job_name: 'athens-proxy'
+      static_configs:
+          - targets: ['athens-proxy:3000']
```

Comment:
a persistent athens-proxy service doesn't seem to be part of the docker-compose itself so I wasn't sure I was supposed to make Prometheus monitor it. But I made Prometheus be able to query athens-proxy by running athens-proxy in the same docker network (athens_default) . I am not sure if I should do this.

Reply:
>I made Prometheus be able to query athens-proxy by running athens-proxy in the same docker network (athens_default)

@vaibhavjayaraman I think that's a great start! We can add more tests to look for specific metrics as we go 😄